controller/network/network.go

```diff
@@ -19,6 +19,11 @@ package network
 import (
```

Comment:
reshuffled imports are ok, as long as they make some kind of sense, if import organization changes are the only changes in the file, please revert, just to minimize churn :)

Reply:
Can do! Wasnt sure so left it in anway